tests/TestCase/Database/QueryTest.php

```diff
@@ -2227,21 +2227,6 @@ public function testSelectDistinctON()
             [3, 1],
             collection($results)->sortBy('author_id')->extract('author_id')->toList()
         );
-
-        $query = new Query($this->connection);
```

Comment:
The exact same query instance and assertions are done above. So unless I am missing something the removed code was redundant.